.circleci/config.yml

```diff
@@ -350,7 +452,9 @@ jobs:
     steps:
       - checkout
       - attach_workspace: *attach_workspace
-      - *restore_node_modules
+      - *restore_root_node_modules
+      - *restore_devtools_node_modules
```

Comment:
I don't think we need to restore DevTools modules for most of these CI tasks, do we?

Just `build_devtools_and_process_artifacts` and `yarn_test_build_devtools`?

(And maybe we don't need to `restore_package_node_modules` for those two tasks either?)

Reply:
Hmm, I wasn't sure what was and wasn't needed by the various CI tasks, so I figured it'll be safest to just restore everything. I'd expect e.g. the build, test, flow tasks to need everything restored?

Either way, the devtools and package caches are really cheap to restore (~0s) so maybe it won't be worth checking if they're necessary?

![image](https://user-images.githubusercontent.com/12784593/92133031-7d8b0c80-ee3a-11ea-907b-70ba71aa9410.png)


In any case, if we use the consolidated node_modules cache approach in my comment above we can go back to having a single cache :D